src/shared/netif-naming-scheme.c

```diff
@@ -13,6 +13,7 @@ static const NamingScheme naming_schemes[] = {
         { "v243", NAMING_V243 },
         { "v245", NAMING_V245 },
         { "v247", NAMING_V247 },
+        { "v248", NAMING_V248 },
```

Comment:
we are too close to the v248 release, please move to v249

Reply:
Will do.